<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame {
	background-color: white;
}

#title {
	font-size: 12pt;
	font-weight: bold;
	margin: 20px 20px 20px 20px;
}

#overview_frame {
	background-color: #f0fafb;
}

#difficulty_label, #score_label, #time_label {
	font-size: 12pt;
	color: #333333;
}

#difficulty_float, #score_int, #time_val {
	font-size: 11pt;
	color: #048c77;
	font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Quiz Overview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="overview_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="difficulty_label">
           <property name="text">
            <string>Average Difficulty</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="score_int">
           <property name="text">
            <string>5 / 5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="time_label">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="score_label">
           <property name="text">
            <string>Score</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="difficulty_float">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="time_val">
           <property name="text">
            <string>12:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_btn">
        <property name="text">
         <string>Back to Quiz Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
